frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineRunItem.tsx

```diff
@@ -27,11 +28,15 @@ const PipelineRunItem: React.FC<PipelineRunItemProps> = ({ pipelineRun }) => {
   const logDetails = getLogSnippet(pipelineRun);
 
   return (
-    <li className="list-group-item">
+    <li className="odc-pipeline-run-item list-group-item">
       <Grid hasGutter>
         <GridItem span={6}>
           <Link to={`${path}`}>{name}</Link>
-          {lastUpdated && <span className="text-muted">&nbsp;({fromNow(lastUpdated)})</span>}
+          {lastUpdated && (
+            <span className="odc-pipeline-run-item__time text-muted">
+              &nbsp;({fromNow(lastUpdated)})
```

Comment:
We shouldn't be using `&nbsp;` anymore because when the time wraps, it creates an unwanted space offset at the beginning of the line.

Reply:
removed nbsp from both Pipeline and Build